app/assets/stylesheets/mailer.scss

```diff
@@ -27,6 +27,10 @@
   }
 }
 
+p, h1 {
+  margin: 0px;
```

Comment:
you don't need use 'px' for zero values
I think you your css libraries already have classes for margin: 0 and padding: 0, you can check it and just add them to template

Reply:
why global?

now all h1 and p have 0 margin.

Do it only for specific place